django/contrib/auth/management/__init__.py

```diff
@@ -66,7 +66,7 @@ def create_permissions(app_config, verbosity=2, interactive=True, using=DEFAULT_
     except LookupError:
         return
 
-    if not router.allow_migrate(using, Permission):
+    if not router.allow_migrate(using, 'auth', Permission):
```

Comment:
I'm wary of making API changes, especially obscure ones, in reaction to another change that had unintended effects. Can we step back, look at the whole picture, and discuss what the API for allow_migrate should be (preferrably on django-developers)?